python/get_config.py

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/python -B
+
+# Insert the parent directory into the module import search path.
+import os
+import sys
+sys.path.insert(1, os.path.dirname(sys.path[0]))
+
+import argparse
+import errno
+import code
+import struct
+import time
+from taulabs import uavo, telemetry, uavo_collection
```

Comment:
There should be a blank line before this one (separate standard modules from others)